### DISCUSSION TITLE
arm64 image builds?

### DISCUSSION THREAD

**Discussion by hemna** [tiangolo/uvicorn-gunicorn-starlette-docker#184] [Questions] [answered]
Can you add image builds for the arm64 architecture?

**Comment by tiangolo** [comment, marked as answer]
Now that Uvicorn supports managing workers with `--workers`, including restarting dead ones, there's no need for Gunicorn. That also means that it's much simpler to build a Docker image from scratch now, I updated the docs to explain it.

Because of that, I deprecated this Docker image: https://github.com/tiangolo/uvicorn-gunicorn-starlette-docker#-warning-you-probably-dont-need-this-docker-image

That also means that if you build your Docker image from the official Python base, you would have multi-arch by default. :sunglasses:
